gsoc.yml

```diff
@@ -58,7 +58,7 @@ gsoc2019:
     rss_feed: "https://aquatiko.github.io/series/gsoc/index.xml"
     project: 'JuliaAstro'
   jredondopizarro:
-    rss_feed: "https://medium.com/feed/@jrpg1996" 
+    rss_feed: "https://medium.com/feed/@jrpg1996"
```

Comment:
@theand9 careful with what you editor changes automatically. It's not a good practice to commit changes that are not related to what you are suggesting to change.

Reply:
Yes sorry will make sure next time